tools/add_triage_signature.py

```diff
@@ -257,6 +268,60 @@ def _update_ticket(self, url, issue_key, should_update=False):
         self._update_triaging_ticket(issue_key, report, should_update=should_update)
 
 
+class InstallationDiskFIOSignature(Signature):
+    def __init__(self, jira_client):
+        super().__init__(jira_client, comment_identifying_string="h1. Host installation disk fio:")
+
+    @staticmethod
+    def _get_fio_events(events):
+        fio_regex = re.compile(r'\(fdatasync duration:\s(\d+)\sms\)')
+        fio_duration_events = ((event, next(map(int, fio_regex.findall(event["message"])), None))
+                               for event in events)
+        fio_duration_events = [(event, fio_duration) for event, fio_duration in fio_duration_events
+                               if fio_duration is not None]
+
+        return fio_duration_events
+
+    def _update_ticket(self, url, issue_key, should_update=False):
+
+        url = self._logs_url_to_api(url)
+        try:
+            md = get_metadata_json(url)
+        except Exception as e:
+            logger.info("Error getting logs for %s at %s: %s, they may have been deleted", issue_key, url, e)
+            return
+
+        cluster = md['cluster']
+
+        cluster_id = cluster['id']
+        events = get_events_json(cluster_url=url, cluster_id=cluster_id)
+        fio_events = self._get_fio_events(events)
+
+        fio_events_by_host = defaultdict(list)
+        for event, fio_duration in fio_events:
+            fio_events_by_host[event["host_id"]].append((event, fio_duration))
+
+        hosts = []
+        for host in cluster['hosts']:
+            host_fio_events = fio_events_by_host[host["id"]]
+            fio_message = "Under threshold"
+            if len(host_fio_events) != 0:
```

Comment:
```python
if host_fio_events:
```

Reply:
That would work, but since it can mean a lot of other things (is host_fio_events potentially None? can it be maybe False?). The current form emphasizes it being a list/tuple and clarifies my expectations of it (being non-empty)